crates/exex/src/hints.rs

```diff
@@ -110,85 +94,6 @@ impl Hint {
-         -> HintExecutionResult {
-            // Open the SQLite database connection.
-            let connection = Connection::open(DATABASE_PATH)
-                .map_err(|e| HintError::CustomHint(e.to_string().into_boxed_str()))?;
-
-            // Initialize the database with the connection.
-            let db = Database::new(connection)
-                .map_err(|e| HintError::CustomHint(e.to_string().into_boxed_str()))?;
-
-            // Retrieve the latest block from the database.
-            let latest_block = db
-                .latest_block()
-                .map_err(|e| HintError::CustomHint(e.to_string().into_boxed_str()))?;
-
-            // If a block was found, print each transaction hash.
-            if let Some(block) = latest_block {
-                for tx in &block.body {
-                    println!("Block: {}, transaction hash: {}", block.number, tx.hash());
-                }
-            }
-
-            Ok(())
-        },
-    )
-}
-
-/// Generates a hint to store block information in the `Environment` model.
-pub fn block_info_hint(block: SealedBlock, transaction: TransactionSignedEcRecovered) -> Hint {
-    Hint::new(
-        String::from("block_info"),
-        move |vm: &mut VirtualMachine,
-              _exec_scopes: &mut ExecutionScopes,
-              ids_data: &HashMap<String, HintReference>,
-              ap_tracking: &ApTracking,
-              _constants: &HashMap<String, Felt252>|
-              -> HintExecutionResult {
-            // We retrieve the `env` pointer from the `ids_data` hashmap.
-            // This pointer is used to store the block-related values in the VM.
-            let env_ptr = get_ptr_from_var_name("block_info", vm, ids_data, ap_tracking)?;
-
-            // We load the block-related values into the VM.
-            //
-            // The values are loaded in the order they are defined in the `Environment` model.
-            // We start at the `env` pointer.
-            vm.load_data(
-                env_ptr,
-                &[
-                    MaybeRelocatable::from(Felt252::from_bytes_be_slice(&block.beneficiary.0 .0)),
-                    MaybeRelocatable::from(Felt252::from(block.timestamp)),
-                    MaybeRelocatable::from(Felt252::from(block.number)),
-                    MaybeRelocatable::from(Felt252::from_bytes_be_slice(&block.mix_hash.0[16..])),
-                    MaybeRelocatable::from(Felt252::from_bytes_be_slice(&block.mix_hash.0[0..16])),
-                    MaybeRelocatable::from(Felt252::from(block.gas_limit)),
-                    MaybeRelocatable::from(Felt252::from(
-                        transaction.chain_id().unwrap_or_default(),
-                    )),
-                    MaybeRelocatable::from(Felt252::from(
-                        block.base_fee_per_gas.unwrap_or_default(),
-                    )),
-                ],
-            )
-            .map_err(HintError::Memory)?;
-
-            Ok(())
-        },
-    )
-}
```

Comment:
We don't need this hint anymore? It was linked to a very recent issue here no? https://github.com/kkrt-labs/keth/issues/23 (the last one I did if I remember well)

Reply:
no, cf previous answer. you can recylce some part I guess, but the actual hint is not this one (see corresponding block hint in the PR)

Reply:
Ok will recycle and modify this in the follow up Pr after we merge this one